selendroid-common/src/main/java/io/selendroid/common/SelendroidCapabilities.java

```diff
@@ -123,7 +123,7 @@ public String getLocale() {
     return (String) getRawCapabilities().get(LOCALE);
   }
 
-  public Map<String, Object> getRawCapabilities() {
+  private Map<String, Object> getRawCapabilities() {
```

Comment:
Just something small i noticed while i was in this class.  I couldnt find anything else in selendroid referencing this method so I thought maybe it was set as public on accident.  BUT you just made me realize, what if an end user is using this method instead of the getters for the specific selendroid capability they are querying.

Let me change it back to public since i dont want to break an external user that might be using this method already public.